content/02.delete-me.md

```diff
@@ -10,6 +10,10 @@ Compare the input (`.md` files in the `/content` directory) to the output you se
 
 [Semi-bold text]{.semibold}
 
+[Centered text]{.center}
```

Comment:
@vincerubinetti can you look over discussion at https://github.com/jgm/pandoc/issues/719 to make sure that we're approaching this inline with any guidance from pandoc maintainers.

Reply:
As best I can tell, they haven't resolved that yet. In lieu of any markdown support for text alignment, the only other option is to use CSS classes, as I've done. It looks like for some reason alignment was considered "styling" and outside of the scope of Markdown (which I disagree with, but here we are).